virttest/utils_test/libvirt.py

```diff
@@ -3611,8 +3611,14 @@ def customize_libvirt_config(params,
     :return: utils_config.LibvirtConfigCommon object
     """
     # Hardcode config_type to virtqemud under modularity daemon mode when config_type="libvirtd"
+    # On the contrary, hardcode it to "libvirtd" when config_type="virt*d".
```

Comment:
Maybe it would be more clear if move the above two comment lines into corresponding if/else body

Reply:
I've updated the code's order as the comment. I think it should be better now.